ncc/examples/monogram.c

```diff
@@ -0,0 +1,3536 @@
+#include <uvm/syscalls.h>
+#include <uvm/utils.h>
+
+
+// Monogram 12x7 pixel font
+
+// > # CREDITS
+// > 
+// > Monogram is a free and Creative Commons Zero pixel font,
+// > made by Vinícius Menézio (@vmenezio).
+// > 
+// > https://datagoblin.itch.io/monogram
+// > 
+// > 
+// > # SPECIAL THANKS
+// > 
+// > thanks to Ateş Göral (@atesgoral) for creating the bitmap font converter:
+// > https://codepen.io/atesgoral/details/RwGOvPZ
+// > 
+// > thanks to Éric Araujo (@merwok_) for the inital port of monogram to PICO-8:
+// > https://itch.io/post/2625522
+
+u64 font_monogram_number_of_characters = 390;
+u64 font_monogram_height = 12;
+u64 font_monogram_width = 7;
+
+u8 font_monogram_data[390][12];
+
+// Map from ord(ch) to index of ch's data in font_monogram_data.
+// This is for when you know the character that you want to draw but not
+// its index in the font_monogram_data array.
+u16 font_monogram_chars[8595];
```

Comment:
I'm a bit confused by what this does? It maps from UTF character indices into the data array?

Too used to working with ASCII I guess 😅

Reply:
Yeah, this might be YAGNI, but that's exactly it.  I figure the order of the characters in the character data is unimportant, clients of this code shouldn't need to know it.  You would use something like `font_monogram_data[font_monogram_chars[ch]]` where `ch` is the UTF "codepoint" (or whatever it's called.)